ipaclient/plugins/ca.py

```diff
@@ -34,13 +35,11 @@ def forward(self, *keys, **options):
         result = super(WithCertOutArgs, self).forward(*keys, **options)
         if filename:
             if options.get('chain', False):
-                certs = (x509.load_der_x509_certificate(c)
-                         for c in result['result']['certificate_chain'])
+                certs = result['result']['certificate_chain']
             else:
-                certs = [
-                    x509.load_der_x509_certificate(
-                        result['result']['certificate'])
-                ]
+                certs = [result['result']['certificate']]
+            certs = (x509.load_der_x509_certificate(base64.b64decode(cert))
```

Comment:
Tricky! This approach is a bit of a hack. In Python 3, ``base64.b64decode()`` accepts both ASCII text and ASCII bytes, and returns bytes. It works for me.